packages/toolpad-app/src/toolpad/AppEditor/PageEditor/RenderPanel/RenderOverlay.tsx

```diff
@@ -965,83 +955,138 @@ export default function RenderOverlay({ canvasHostRef }: RenderOverlayProps) {
       const isDraggingOverLayoutSlot = dragOverSlot?.type === 'layout';
       const isDraggingOverElement = appDom.isElement(dragOverNode);
 
-      domApi.update((draft) => {
-        let parent = appDom.getParent(draft, dragOverNode);
+      domApi.update(
```

Comment:
This is all just formatting changes except the second argument passed to `domApi.update`.